app/views/csv_exports/new_deploy_group_usage.erb

```diff
@@ -0,0 +1,16 @@
+<%= page_title "Deploy Group Usage Report" %>
+
+<section>
+
+  <div class="csv-users-filters clearfix">
+    <%= form_tag new_csv_export_path(format: :csv, type: :deploy_group_usage), method: :get, class: "form-horizontal" do %>
+      <%= hidden_field_tag :type, "deploy_group_usage" %>
```

Comment:
it's in the url so no need for it here ?

Reply:
When I remove `hidden_field_tag` the csv generation+download fails - seems to be part of the weird html and csv routing in the `csv_exports_controller` that makes this dance work. I poked around a little but couldn't find a way to remove it - however since this is the same pattern employed in `new_users.html.erb` - hopefully not too egregious to leave it as-ss.